app/serializers/organization-cleanup-preference.js

```diff
@@ -0,0 +1,5 @@
+import DRFSerializer from './drf';
+
+export default DRFSerializer.extend({
+  primaryKey: 'is_enabled'
```

Comment:
wait...why is the primaryKey is_enabled?

Reply:
There is no id in the response payload so, model couldn't sync it.